src/msgpack.d

```diff
@@ -2760,6 +2748,53 @@ struct Unpacker
     }
 
 
+    void unpackObject(T)(ref T object) if (is(Unqual!T == class))
```

Comment:
s/is(Unqual!T == class))/is(T==class)
Unqual seems un-necessary:
assert(is(Unqual!(const(Object)) == class)); // passes